{"name": "Принять заказ. Попытка принять заказ с некорректным id курьера возвращает корректное сообщение об ошибке", "status": "passed", "description": "тест ручки \"Принять заказ\"", "start": 1724853724484, "stop": 1724853724670, "uuid": "1b32aa72-05fd-4266-97bc-74e18f41008f", "historyId": "dc56c8611b5ea8558e0798144e5a2c53", "testCaseId": "dc56c8611b5ea8558e0798144e5a2c53", "fullName": "tests.TestAcceptOrder#test_try_accept_order_with_non_existing_courier_id", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "9332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}